### PR TITLE
Support ACPI events for GPIO 7 interrupt

### PR DESCRIPTION
**Pull request by asmaamellanox**
On the Bluesphere, HOST_GPIO7 is setup as an interrupt
and causes soft reset. This is all handled by the acpid.
bfauxpwr script was used for the ocp3.0 low power interrupt
but it is now utilized for both the low power and reset
interrupt.

### CODE REVIEW

File: man/bfacpievt.8 [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+.SH DESCRIPTION
+Creates files needed to handle low power mode or reset acpi events. This script
+is only applicable for the OCP 3.0 card and the BlueSphere. Called with no
+arguments, this script will do nothing.
+.SH OPTIONS
+.IP --update-acpi-low-pwr
+Disable power button event handling in systemd, and configure the ACPI daemon
+to handle AUX power events by powering up/down all secondary cores. This option
+is only valid for the OCP 3.0 card.
+.IP --update-acpi-rst
```
Comment (lsun100) [inline man/bfacpievt.8:20]: Similarly, "--update-reset"?

File: README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ A public collection of useful scripts to help manage the Mellanox BlueField
 SoC.
 
 Overview of each file:
-- **bfauxpwr** Config ACPI daemon to handle AUX power mode event.
```
Comment (lsun100) [inline README.md:8]: might need another commit in ezchip repo as well to fix the recipe file, or else master branch could fail.

File: man/bfacpievt.8 [hunk 3]
```diff
@@ -0,0 +1,23 @@
+.TH BFAUXPWR 8 "June 2020"
+.SH NAME
+bfacpievt \- Create files needed to handle low power mode or reset acpi event.
```
[the remark this answers is not in the document]
Comment (asmaamellanox) [reply man/bfacpievt.8:3]: done

File: man/bfacpievt.8 [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+.SH SYNOPSIS
+.B bfacpievt
+.RB [ \-\-help | \-h ]
+.PP
+.B bfacpievt --update-acpi-low-pwr
+.PP
+.B bfacpievt --update-acpi-rst
+.SH DESCRIPTION
+Creates files needed to handle low power mode or reset acpi events. This script
+is only applicable for the OCP 3.0 card and the BlueSphere. Called with no
```
[the remark this answers is not in the document]
Comment (asmaamellanox) [reply man/bfacpievt.8:13]: done

File: man/bfacpievt.8 [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+arguments, this script will do nothing.
+.SH OPTIONS
+.IP --update-acpi-low-pwr
```
[the remark this answers is not in the document]
Comment (asmaamellanox) [reply man/bfacpievt.8:16]: done

File: man/bfacpievt.8 (hunk 1 above)
Comment (asmaamellanox) [reply man/bfacpievt.8:20]: done

File: README.md (hunk 2 above)
Comment (asmaamellanox) [reply README.md:8]: whoops I will do it now.

**Review by shravankumarr — COMMENTED** [review]
LGTM

**Pull request merged**